.github/workflows/build.yml

```diff
@@ -55,18 +49,103 @@ jobs:
             protobuf-c-compiler
             tcl
             uuid-dev
-            zlib1g-dev &&
+            zlib1g-dev
+        '
+      - name: Build from source
+        run: '
           (
-            mkdir original_comdb2/build &&
-            cd original_comdb2/build &&
+            mkdir bloomberg-comdb2/build &&
+            cd bloomberg-comdb2/build &&
             cmake .. &&
-            make &&
-            sudo make install
+            make
           )
         '
+      - name: Archive bloomberg-comdb2 repo with build artifacts
+        run: 'tar czvf bloomberg-comdb2.tar.gz bloomberg-comdb2/'
+      - name: Upload bloomberg-comdb2 repo with build artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: bloomberg-comdb2
+          path: ./bloomberg-comdb2.tar.gz
+
+  build_sdist:
+    name: Build python-comdb2 source distribution
+    runs-on: ubuntu-latest
+    needs: [build_bloomberg_comdb2]
+    steps:
+      - name: Download bloomberg-comdb2 with build artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: bloomberg-comdb2
+          path: .
+      - name: Install bloomberg-comdb2
+        run: '
+          sudo apt-get install -qy
+            libevent-dev
+            liblz4-dev
+            libprotobuf-c-dev
+            libsqlite3-dev
+            libssl-dev
+            libunwind-dev
+            zlib1g-dev &&
+          tar xvf bloomberg-comdb2.tar.gz &&
+          (cd bloomberg-comdb2/build && sudo make install) &&
+          rm -rf bloomberg-comdb2.tar.gz bloomberg-comdb2/
```

Comment:
What's the rationale for removing these? It doesn't seem harmful, and it's probably fast - but it doesn't seem helpful either, AFAICS. Is there a reason it's useful that I'm not seeing?

Reply:
No real reason, just that we don't need them. I can remove the `rm` command as this will be less code to maintain.

Reply:
I don't think it _really_ matters either way - I was just wondering if there was a reason why it _must_ be done that I wasn't seeing. If not, your call on whether to keep it or remove it.